templates/BuildMyCharm/config.xml

```diff
@@ -62,13 +63,31 @@ juju add-model test-{{charmname}}-$BUILD_NUMBER
 
 if [ -z "{{referencebundle}}" ]
 then
-    bundletester -e {{controller}}:test-{{charmname}}-$BUILD_NUMBER \
-      -vFt /tmp/$OUTPUT_DIR/{{charmname}}
+    rm -f totest.yaml
+    echo "bundle: /tmp/bundles/{{bundle_fname}}" &gt;&gt; totest.yaml
+    echo "bundle_name: {{charmname}}" &gt;&gt; totest.yaml
+    cwr {{controller}}:test-{{charmname}}-$BUILD_NUMBER totest.yaml --results-dir /srv/artifacts --test-id $BUILD_NUMBER
```

Comment:
I still get this error from cwr:

+ cwr lxd:test-apache-tomee-5 totest.yaml --result-output /srv/artifacts --test-id 5
2017-01-02 15:43:13 INFO Cloud Weather Report started.
2017-01-02 15:43:13 INFO Running test on lxd.
No fetcher for url: /tmp/bundles/
Test result:
  results/_tmp_bundles_-2017-01-02T15:43:13-result.html


This jenkins job needs to build the bundle locally and test it. Is it possible these changes are intended for the RunCwr jenkins job only? I thought cwr would not test local charms.